dxram/src/main/java/com/yahoo/ycsb/db/YCSBObjectPool.java

```diff
@@ -0,0 +1,40 @@
+package com.yahoo.ycsb.db;
+
+/**
+ * Pool for YCSBObjects to avoid allocations.
+ *
+ * @author Stefan Nothaas, stefan.nothaas@hhu.de, 09.11.2018
```

Comment:
denote authorship through git commit message ([use the github feature for multiple authors if needed](https://help.github.com/en/articles/creating-a-commit-with-multiple-authors))